LICENSE.txt

```diff
@@ -0,0 +1 @@
+see COPYING
```

Comment:
The licence is spread across two files, COPYING and COPYING.LESSER, because this project is dual-licensed under GPLv3 and LGPLv3.  This is because LGPLv3 is a modification to GPLv3 that incorporates it by reference.  I think it makes most sense here for the LICENSE file to describe the licence and refer to both of the COPYING files.